libr/anal/fcn.c

```diff
@@ -237,11 +266,12 @@ static void _fcn_tree_iter_next(FcnTreeIter *it, ut64 from, ut64 to) {
 		}
 		x_ = it->path[--it->len];
 		x = FCN_CONTAINER (x_);
-		if (to - 1 < x->addr) {
+		set_meta_min_if_needed (x);
```

Comment:
why? it's weird you need to adjust nodes while iterating over them.

Reply:
same as https://github.com/radare/radare2/pull/13193/files#r262920262

Reply:
see https://github.com/radare/radare2/pull/13193#discussion_r262942662